models/search_result.coffee

```diff
@@ -38,6 +38,8 @@ module.exports = class SearchResult extends Backbone.Model
       "/show/#{@get('id')}"
     else if @get('model') is 'sale'
       "/auction/#{@get('id')}"
+    else if @get('model') is 'city'
+      "/shows/#{@get('id')}"
```

Comment:
Looks like the `id` I got back from the endpoint isn't in the format of `new-york-ny-usa`. Or maybe I missed something here?

https://api.artsy.net/api/v1/match?term=new%20york

<img width="617" alt="screen shot 2016-11-29 at 3 56 30 pm" src="https://cloud.githubusercontent.com/assets/796573/20728614/8ce44d24-b64c-11e6-907c-31c23ecd3aa6.png">